backend/api/models/models.py

```diff
@@ -205,6 +205,12 @@ class WatchlistItem(Base):
 
     company = relationship("Company")
 
+class NewsLetter(Base):
+    __tablename__ = "newsletter"
+
+    id = Column("newletter_id", Integer, primary_key=True, index=True, autoincrement=True)
+    email = Column(String(64))
+    subscribed = Column(Boolean, default=False)
```

Comment:
uhm, what is the use of the **subscribed** field?

Reply:
To track the if the user is subscribed

Reply:
No bro, it shouldn't be there. Subscription verification should be independent of this model